cpp/src/dbscan/vertexdeg/algo.h

```diff
@@ -21,7 +21,6 @@
 #include <common/device_buffer.hpp>
 #include <distance/epsilon_neighborhood.cuh>
 #include "cuda_runtime.h"
-#include "cuda_utils.h"
```

Comment:
@teju85 , is there any reason for this to be present here? 
I have seen such include even in pca.h, and with that, I see pca.cu could be modified to pca.cpp as it seems like just calls to functions and I dont see any device functions.

Reply:
probably an oversight. Makes sense to remove this.